Upgrade to Composable Kernel 7.2 (#396)
amd/rocm-examples: Libraries/ComposableKernel/gemm/flatmm/README.md

## Description

> This example demonstrates how to perform a FLATMM operation with the CK Tile programming model. Multiple variants are provided:

Reviewer: ```suggestion
This example demonstrates how to perform a FLATMM operation using CK Tile. Multiple variants are provided:
```

I left the "with" elsewhere but I think "using" is better
